utils/ansible-runner-service.sh

```diff
@@ -414,17 +414,20 @@ check_access() {
 }
 
 manage_ansible_hosts_file() {
-    echo "Linking the default ansible hosts file to the runner-service inventory."
+    echo "Linking the runner service inventory to ceph-ansible hosts"
+    ln -sf /usr/share/ceph-ansible/hosts /usr/share/ansible-runner-service/inventory/hosts
```

Comment:
this hosts file may not exist - so you may need to check that it's there and if not touch it